CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v2.11.1
```

Comment:
I think we should go with v2.12.0 - we don't have only fixes and patches, but also added new functionality